test/api/v3/integration/inbox/GET-inbox_messages.test.js

```diff
@@ -45,4 +45,23 @@ describe('GET /inbox/messages', () => {
     expect(messages[2].text).to.equal('second');
     expect(messages[3].text).to.equal('first');
   });
+
+  it('returns no messages when using page-query ', async () => {
```

Comment:
can you clarify the text here? I mean why it says returns no messages when in fact 4 are returned?

Reply:
I had the best first just without results, then I thought It would make more sense to have some entries so I changed the test, gonna rename the test now